tests/manage/mcg/test_multi_region.py

```diff
@@ -0,0 +1,141 @@
+import logging
+from itertools import chain
+from random import randrange
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1, aws_platform_required, filter_insecure_request_warning
+)
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.constants import BS_AUTH_FAILED, BS_OPTIMAL
+from tests.helpers import create_unique_resource_name
+from tests.manage.mcg.helpers import retrieve_test_objects_to_pod, sync_object_directory
+
+logger = logging.getLogger(__name__)
+
+
+@filter_insecure_request_warning
+@aws_platform_required
+@tier1
+class TestMultiRegion:
+    """
+    Test the multi region functionality
+    """
+    @pytest.fixture()
+    def resources(self, request, mcg_obj):
+        bs_objs, bs_secrets, bucketclasses, aws_buckets = (
+            [] for _ in range(4)
+        )
+
+        # Cleans up all resources that were created for the test
+        def resource_cleanup():
+            for resource in chain(bs_secrets, bucketclasses):
+                resource.delete()
+
+            for backingstore in bs_objs:
+                backingstore.delete()
+                mcg_obj.send_rpc_query('pool_api', 'delete_pool', {'name': backingstore.name})
+
+            for aws_bucket in aws_buckets:
+                mcg_obj.aws_s3_resource.Bucket(aws_bucket).objects.all().delete()
+                mcg_obj.aws_s3_resource.Bucket(aws_bucket).delete()
+
+        request.addfinalizer(resource_cleanup)
+
+        return aws_buckets, bs_secrets, bs_objs, bucketclasses
+
+    @pytest.mark.polarion_id("OCS-1298")
```

Comment:
this polarion id is already used in tests/manage/mcg/test_bucket_creation.py:20

Reply:
Fixed :)